src/styles/card-block.scss

```diff
@@ -0,0 +1,108 @@
+.main {
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px 0 72px;
+}
+
+.card {
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+
+    &-picture {
```

Comment:
I guess you may get rid of `card__image` wrapper if just set margin and correct size here (for `img` directly)